papers/doc/6-sample/README.md

```diff
@@ -6,14 +6,25 @@ In Furnace, these samples can be generated by importing a .wav (think of it as a
 
 ## supported systems
 
-As of Furnace 0.5.5, the following sound chips have sample support:
- - NES/Ricoh 2A03 (PCM only, no DPCM, and only on channel 5)
- - Sega Genesis/YM2612 (channel 6 only; but only if there exists a `1701` effect that gets played on or before a trigger for a sample)
+As of Furnace dev99, the following sound chips have sample support:
+ - NES/Ricoh 2A03 (with DPCM support and only on channel 5)
+ - Sega Genesis/YM2612 (channel 6 only; but only if there exists a `1701` effect that gets played on or before a trigger for a sample, or if you are using an instrument with Sample type)
  - PC Engine/TurboGrafx 16/Huc6280 (same conditions as above)
- - Amiga/Paula (on all channels AND resamplable, but you need to make an instrument with the Amiga format and tie it to a sample first)
- - Arcade/SEGA PCM (same as above but you don't need to make an instrument for it and you have to use the `20xx` effect command to resample your samples)
- - Neo Geo/Neo Geo EXT-Ch2 (on the last 7 channels only and can be resampled the same way as above)
- - Seta/Allumer X1-010 (same as above, and both `1701` and `20xx` effect commands are affected on all 16 channels)
+ - Amiga/Paula (on all channels)
+ - Arcade/SEGA PCM (same as above)
+ - Neo Geo/Neo Geo CD (on the last 7 channels (6 if you are using Neo Geo CD) only and can be resampled the same way as above)
+ - Seta/Allumer X1-010 (same as YM2612)
+ - Atari Lynx (though the option is a bit hidden)
```

Comment:
>   - Atari Lynx (though the option is a bit hidden)

What do you mean? It operates via Sample instrument type out of the box

Reply:
hm, I guess I never noticed that. will remove that shortly